test/test_core.py

```diff
@@ -2713,6 +2713,14 @@ def test_atexit_threads(self):
     self.set_setting('EXIT_RUNTIME')
     self.do_core_test('test_atexit_threads.cpp')
 
+  @node_pthreads
+  def test_pthread_cancel(self):
+    self.do_run_in_out_file_test('pthread/test_pthread_cancel.cpp')
```

Comment:
This file already existed but wasn't being run? I don't see it in the PR.

Reply:
This was a previously only run as  part of the browser tests.  We run a subset of pthread tests as both node and browser tests.